[2024-11-03T22:42:22.575+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T22:42:22.589+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: self-tutorial-dag.start_task scheduled__2024-11-02T00:00:00+00:00 [queued]>
[2024-11-03T22:42:22.597+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: self-tutorial-dag.start_task scheduled__2024-11-02T00:00:00+00:00 [queued]>
[2024-11-03T22:42:22.597+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-03T22:42:22.611+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): start_task> on 2024-11-02 00:00:00+00:00
[2024-11-03T22:42:22.616+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=75) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T22:42:22.618+0000] {standard_task_runner.py:72} INFO - Started process 79 to run task
[2024-11-03T22:42:22.618+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'self-tutorial-dag', 'start_task', 'scheduled__2024-11-02T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/self-tutorial-dag.py', '--cfg-path', '/tmp/tmpi_czwa6x']
[2024-11-03T22:42:22.618+0000] {standard_task_runner.py:105} INFO - Job 92: Subtask start_task
[2024-11-03T22:42:22.663+0000] {task_command.py:467} INFO - Running <TaskInstance: self-tutorial-dag.start_task scheduled__2024-11-02T00:00:00+00:00 [running]> on host 2839f63d218a
[2024-11-03T22:42:22.743+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='self-tutorial-dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-02T00:00:00+00:00'
[2024-11-03T22:42:22.744+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T22:42:22.756+0000] {python.py:240} INFO - Done. Returned value was: About to fall a sleep
[2024-11-03T22:42:22.777+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T22:42:22.777+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=self-tutorial-dag, task_id=start_task, run_id=scheduled__2024-11-02T00:00:00+00:00, execution_date=20241102T000000, start_date=20241103T224222, end_date=20241103T224222
[2024-11-03T22:42:22.792+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T22:42:22.810+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-03T22:42:22.813+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
